tools/tokenserver/util.py

```diff
@@ -56,3 +97,25 @@ def format_key_id(keys_changed_at, key_hash):
 def get_timestamp():
     """Get current timestamp in milliseconds."""
     return int(time.time() * 1000)
+
+
+class Metrics():
+    prefix = ""
+    client = None
+
+    def __init__(cls, opts):
+        options = dict(
+            namespace=getattr(opts, "app_label", ""),
+            statsd_namespace=getattr(opts, "app_label", ""),
```

Comment:
I dunno. `util` is a library. It's used by a bunch of scripts. We can either ensure that all the callers provide that argument or be fine with those scripts breaking, or we can just see if it's defined in the object that's passed. I think I prefer this approach better since there's less chance of breaking random crap.

Reply:
We could also accomplish the same by making `app_label` kwarg with a default of "" (I'd probably just edit all the scripts since it's a couple 2 liners, but there are a number of them so it's annoying -- we should probably consider removing many of them since they're rarely used and our recent actions of calling something such as e.g. 'update_node' is just SRE executing statements directly but I digress..)

I won't die on this hill but my preference for not involving optparse is it suggests to readers of the code that the intention of this argument is to be overwritten on the command line, when that's never going to (and shouldn't) happen and in the minute possibility that it does it's not going to be for a good reason.

Reply:
Heh, one of the things I was really trying to avoid was altering util in any way that breaks a script that calls. Possibly futile, but at this point, god only knows what does and I didn't want a bunch of stuff breaking. 

I'm fine making it a kwd.